osgtest/tests/test_150_xrootd.py

```diff
@@ -76,7 +76,7 @@ def test_02_configure_hdfs(self):
         files.append(core.config['xrootd.config'], hdfs_config, backup=False)
 
     def test_03_configure_multiuser(self):
-        core.skip_ok_unless_installed('xrootd-multiuser')
+        core.skip_ok_unless_installed('xrootd-multiuser','globus-proxy-utils', by_dependency=True)
         core.config['xrootd.multiuser'] = "ON"
```

Comment:
We should initially set this to `False` and then `True` at the very end of the test